mailpoet/RoboFile.php

```diff
@@ -153,7 +153,7 @@ public function translationsBuild() {
       ->taskExec("php -d memory_limit=-1 tasks/makepot/makepot-views.php . > lang/mailpoet.pot")
 
       // PHP, JS/TS
-      ->taskExec("vendor/bin/wp i18n make-pot --merge --slug=mailpoet --domain=mailpoet --exclude=$exclude --headers=$headers . lang/mailpoet.pot")
+      ->taskExec("php -d memory_limit=-1 $(which wp) i18n make-pot --merge --slug=mailpoet --domain=mailpoet --exclude=$exclude --headers=$headers . lang/mailpoet.pot")
```

Comment:
@samnajian Hey! Sorry to jump in, I was just browsing through and this caught my eye. Apart from fixing the memory limit, this seems to break the command anywhere outside Docker container by changing `vendor/bin/wp` to `$(which wp)`. Could we revert that change? Thanks!